components/Feature/VulnerabilitiesGrid/index.js

```diff
@@ -165,32 +166,33 @@ const VulnerabilitiesGrid = ({ resources, onUpdate }) => {
                               value: label
                             })
                           }
+                          aria-expanded={
+                            textinputs
+                              ? selectedVulnerabilities
+                              : undefined
+                          }
                         />
                         {textinputs &&
                           grid.vulnerabilities[cbId] &&
                           textinputs.map(({ label, type }) => {
                             const inputId = `${cbId}-${labelToId(label)}-i`;
                             return (
-                              <React.Fragment key={inputId}>
-                                <p id={`${inputId}-aria`} hidden>
-                                  This is a text input for {cbLabel}: {label}
-                                </p>
-                                <TextInput
-                                  name={inputId}
-                                  label={label}
-                                  onChange={value =>
-                                    updateTextData({
-                                      key: inputId,
-                                      value,
-                                      label,
-                                      parentKey: cbId,
-                                      inputType: type,
-                                      gridType: 'vulnerabilities'
-                                    })
-                                  }
-                                  aria={`${inputId}-aria`}
-                                />
-                              </React.Fragment>
+                              <TextInput
+                                key={inputId}
+                                name={inputId}
+                                label={label}
+                                onChange={value =>
+                                  updateTextData({
+                                    key: inputId,
+                                    value,
+                                    label,
+                                    parentKey: cbId,
+                                    inputType: type,
+                                    gridType: 'vulnerabilities'
+                                  })
+                                }
+                                aria={`${inputId}-aria`}
```

Comment:
Did you put the description for this aria somewhere else? It was on lines 175-177 
```suggestion
<p id={`${inputId}-aria`} hidden>
                                   This is a text input for {cbLabel}: {label}
                                 </p>
```